src/textual/geometry.py

```diff
@@ -35,7 +36,7 @@ class Offset(NamedTuple):
 
     @property
     def is_origin(self) -> bool:
-        """Check if the point is at the origin (0, 0)"""
+        """Return True if the point is at the origin (0, 0), else False"""
```

Comment:
I like to avoid this wording, as its verbose. Every bool return would have the tiresome phrasing "Return True if blah else False".